### ISSUE TITLE
If sqlite3 relies on @mapbox/node-pre-gyp, automatically tap into that

### ISSUE THREAD

**Issue by Pomax** [TryGhost/node-sqlite3#1479]
Simply running npm install for the sqlite3 library generates a warning that people have no control over:
```
$ npm install sqlite3
npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
```

If sqlite3 requires a different node-pre-gyp than people will have simply because they use Node, telling them to upgrade isn't going to do much good: upgrade what? upgrade how? And also, upgrade why? They know nothing about node-pre-gyp, "why can't mapbox just update the regular node-pre-gyp library?".

If sqlite3 needs its own custom version of node-pre-gyp, then it should pull that in as late dependency during its own `npm install` pass.

**Comment by inukshuk** [comment]
The dependency here in the repository has already been update, but we're waiting for it to be released to NPM.

**Comment by Pomax** [comment]
roger - thanks!

**Comment by pulli23** [comment]
Any timeframe for when an update is pushed to npm? This is also giving some errors on `npm audit` as node-pre-gyp and node-gyp versions used here use an "ancient" `tar` library. Which result in some high severity errors.

**Comment by daniellockyer** [comment]
This should be fixed with v5.0.3 🙂
